### PR TITLE
ref(feedback-dialog) Update design

### PR DESCRIPTION
**Pull request by robertpin**
Convert file to TS
Move styles from SCSS to JSS

### CODE REVIEW

File: react/features/feedback/components/FeedbackDialog.web.tsx [hunk 1]
```diff
@@ -215,34 +273,35 @@ class FeedbackDialog extends Component<Props, State> {
...
                         { isFilled
-                            ? <StarFilledIcon
-                                label = 'star-filled'
-                                size = 'xlarge' />
-                            : <StarIcon
-                                label = 'star'
-                                size = 'xlarge' /> }
+                            ? <Icon
+                                color = '#1EC26A'
+                                size = { 32 }
```
Comment (saghul) [inline react/features/feedback/components/FeedbackDialog.web.tsx:300]: Extract to a constant?

File: react/features/feedback/components/FeedbackDialog.web.tsx [hunk 2]
```diff
@@ -215,34 +273,35 @@ class FeedbackDialog extends Component<Props, State> {
...
                         }) }>
                         { isFilled
-                            ? <StarFilledIcon
-                                label = 'star-filled'
-                                size = 'xlarge' />
-                            : <StarIcon
-                                label = 'star'
-                                size = 'xlarge' /> }
+                            ? <Icon
+                                color = '#1EC26A'
```
Comment (saghul) [inline react/features/feedback/components/FeedbackDialog.web.tsx:299]: Define as a token pl

**Review by saghul — COMMENTED** [review]
Left a couple of comments

**Comment by robertpin** [comment]
Jenkins test this please

**Review by saghul — APPROVED** [review]

**Pull request merged**